### PR TITLE
WIP: apps/budgeting/proposals/filter: adjusting template

### PR DESCRIPTION
**Pull request by khamui**
to switch from map to list (in django)
to switch from list to map (in react)

### CODE REVIEW

File: meinberlin/apps/budgeting/assets/FilterBar.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react'
```
Comment (khamui) [inline meinberlin/apps/budgeting/assets/FilterBar.jsx:1]: this file was just copied as base for the Filters, is going to be changed entirely.

Comment (phillimorland) [reply meinberlin/apps/budgeting/assets/FilterBar.jsx:1]: @khamui i did a reuseable switch component to replace the switch in react but not sure if it'll work with how we want to do it, but we should talk about it before we go further to not double up work https://github.com/liqd/a4-meinberlin/pull/3981/files

Comment (khamui) [reply meinberlin/apps/budgeting/assets/FilterBar.jsx:1]: @phillimorland true! lets use the switch component if possible :) you want to take a look if its possible?
I will focus on the filters then and leave the button untouched. does that make sense?

**Comment by khamui** [comment]
will implement feature on a new branch.

**Pull request closed without merging**